cmd/ecs-deploy/task_def.go

```diff
@@ -221,7 +226,7 @@ func initTaskDefFlags(flag *pflag.FlagSet) {
 	// Task Definition Settings
 	flag.String(serviceFlag, "app", fmt.Sprintf("The service name (choose %q)", services))
 	flag.String(environmentFlag, "", fmt.Sprintf("The environment name (choose %q)", environments))
-	flag.String(imageURIFlag, "", "The URI of the container image to use in the task definition")
+	flag.String(imageURIFlag, "", "The URI of the container(s) image(s) to use in the task definition")
```

Comment:
It's not multiple containers, just multiple container images. The images themselves turn into multiple containers. So maybe taking out the word `container` would clear things up.

```suggestion
	flag.String(imageURIFlag, "", "The URI of the image(s) to use in the task definition")
```